website/docs/faqs/Git/git-migration.md

```diff
@@ -9,18 +9,19 @@ tags: [Git]
 
 To migrate from one git provider to another, refer to the following steps to avoid minimal disruption:
 
-1. Outside of dbt Cloud, you'll need to import your existing repository into your new provider. 
+1. Outside of dbt Cloud, you'll need to import your existing repository into your new provider. By default, connecting your repository in one account won't automatically disconnected it from another account.
```

Comment:
What does this addition mean, broadly? When I try to connect a GitHub account that is already connected to another dbt Cloud account, it won't let me. If they do manage to get it connected to two accounts somehow, what's the implication?

Reply:
this is more to explicitly inform users that somehow connecting a project/account to a project repo won't disconnect that repo in the other account during a migration.